_data/projects.yml

```diff
@@ -240,6 +248,9 @@
   description: "Operations for primitive and String singleton types"
   github: "https://github.com/fthomas/singleton-ops"
   affiliate: true
+- title: "Skunk"
```

Comment:
Should we add the same category tag as doobie? "Principled database access"?

Reply:
I was thinking about that as well, but then thought the whole list of categories could use a review and didn't.
I might even steal the category label and use it for denoting platform support. I don't think the category label is carrying its weight.

Reply:
Probably worth a conversation about what our taxonomy ought to be, yes.